Makefile

```diff
@@ -1,5 +1,5 @@
 PACKAGE_NAME=github.com/projectcalico/calicoctl
-GO_BUILD_VER=v0.49
+GO_BUILD_VER=add-arm-7
```

Comment:
@aalaesar Looks like v0.52 has the commtis you need in it now? https://github.com/projectcalico/go-build/commits/v0.52

Reply:
Same for the other PRs in this series:
- https://github.com/projectcalico/kube-controllers/pull/640/
- https://github.com/projectcalico/pod2daemon/pull/54/
- https://github.com/projectcalico/cni-plugin/pull/1031

Reply:
Hello @caseydavenport 
thank you for pointing this
I had some issue that kept me away from calico this past month
I'm glad  gobuild got finally the v0,52 tag uploaded  to docker hub.
that's was an event I was waiting for in march
I'll update the makefiles ASAP

Regards